api/controllers/hoard-controller.js

```diff
@@ -15,6 +15,7 @@ const createHoard = (req, res, next) => {
 
   const plaintextIn = {
     data: addMeta(meta, req.files[0].buffer),
+    salt: req.query.salt ? Buffer.from(req.query.salt) : Buffer.from(process.env.HOARD_SALT),
```

Comment:
Do we make use of transmitting a salt value in the query parameters? It seems like (if at all) a custom salt should be transmitted as part of the body in an HTTP post for security reasons. Or does the Hoard endpoint specify this option?